ha-example/README.md

```diff
@@ -0,0 +1,160 @@
+# Flink cluster HA configuration
+
+In the deployment of Flink cluster, it consists of operators, job managers, and task managers.
+By default, the deployment is not a high availability (HA) cluster, we need to configure it to achieve it.
+
+## Flink operator
+
+The Flink operator manages and operates Flink Deployments. It supports high availability through leader election and standby operator instances.
+To enable leader election we need to add the following two mandatory operator configuration parameters.
+
+```
+kubernetes.operator.leader-election.enabled: true
+kubernetes.operator.leader-election.lease-name: flink-operator-lease
+```
+The lease name must be unique in the current lease namespace.
+
+So, when installing flink operator using helm, we need to add these 2 configs + existing default configs via command like this:
+```
+helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator \
+--set podSecurityContext=null \
+--set defaultConfiguration."log4j-operator\.properties"=monitorInterval\=30 \
+--set defaultConfiguration."log4j-console\.properties"=monitorInterval\=30 \
+--set replicas=2 \
+--set defaultConfiguration."flink-conf\.yaml"="kubernetes.operator.metrics.reporter.slf4j.factory.class\:\ org.apache.flink.metrics.slf4j.Slf4jReporterFactory
+kubernetes.operator.metrics.reporter.slf4j.interval\:\ 5\ MINUTE
+kubernetes.operator.reconcile.interval:\ 15\ s
+kubernetes.operator.observer.progress-check.interval:\ 5\ s
+kubernetes.operator.leader-election.enabled:\ true
+kubernetes.operator.leader-election.lease-name:\ flink-operator-lease" \
+-n flink
+```
+Note: The `replicas` config is to set the number of operator replicas.
```

Comment:
```suggestion
Note: The `replicas` configuration is to set to the total number of Flink Kubernetes operator replicas you want.
This will include the leader plus the number of standby instances you want. 
One standby (two replicas in total) is usually sufficient.
```